client/devdocs/gutenberg-components/style.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../../node_modules/@wordpress/components/build-style/style';
+// @import '../../../node_modules/@wordpress/components/build-style/style';
```

Comment:
cc @mmtr since you've worked on this. If I leave the style include here I'm running into:

```
{
  "status": 1,
  "file": ".../wp-calypso/node_modules/@wordpress/components/build-style/style.css",
  "line": 1615,
  "column": 27,
  "message": "Function shade is missing argument $percent.",
  "formatted": "Error: Function shade is missing argument $percent.\n        on line 1 of client/devdocs/gutenberg-components/style.scss\n        from line 7 of assets/stylesheets/sections/devdocs.scss\n>>       background: #0085ba shade(15%); }\n\n   --------------------------^\n"
}
```

The mentioned `shade` function has the argument `percent` passed at that line if you check the built file. Any ideas what might be causing this?

Reply:
I'd say that the `shade` function shouldn't be in the generated CSS file given that it's not a CSS function AFAIK but I can see it on the original [SCSS file](https://github.com/WordPress/gutenberg/blob/61798e98fb37dfd7ee1ef5edab3c29f0fe0d8e1a/packages/components/src/date-time/style.scss#L43) (added on WordPress/gutenberg#7621). I'm sure that function should be converted to something else during the CSS build process.

I see the `shade` function is also used in other components (i.e. [`Button`](https://github.com/WordPress/gutenberg/blob/61798e98fb37dfd7ee1ef5edab3c29f0fe0d8e1a/packages/components/src/button/style.scss#L101)), but in all of them the function is an argument of the `color` function, so maybe the issue is on Gutenberg and it can be solved if we replace `background: theme(primary) shade(15%);` with `background: color(theme(primary) shade(15%));` on the original [SCSS file](https://github.com/WordPress/gutenberg/blob/61798e98fb37dfd7ee1ef5edab3c29f0fe0d8e1a/packages/components/src/date-time/style.scss#L43).

Reply:
Confirmed that will solve the issue. Gutenberg uses [`postcss-color-function`](https://github.com/postcss/postcss-color-function) which provides the `color` interface and `shade` as one of their color adjusters.

I can open a PR fixing that as soon as I got access to the Gutenberg repo :)

Reply:
@gziolo could we add @mmtr to the repo to file the above fix? :)

Reply:
I already asked for that in p1540284879000100-slack-gutenberg. I think only @mtias can add me.

Reply:
Yes, him and @pento :)

Reply:
@mmtr if you don't want to wait feel free to start a PR from your fork too, but we'll definitely want to provide you with access for future fixes.

Also, it's curious that the same import wasn't causing issues for editor shell in Calypso:
https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/sections/gutenberg-editor.scss#L19
That's why I suspected it had something to do with differences in DevDocs setup at first.

Reply:
@vindl Done! https://github.com/WordPress/gutenberg/pull/10943

Reply:
Thanks Miguel!